packages/open-next/src/wrappers/aws-lambda-streaming.ts

```diff
@@ -6,14 +6,30 @@ import { StreamCreator } from "http/index.js";
 import { WrapperHandler } from "types/open-next";
 
 import { error } from "../adapters/logger";
-import { WarmerEvent } from "../adapters/warmer-function";
+import { WarmerEvent, WarmerResponse } from "../adapters/warmer-function";
 
 type AwsLambdaEvent = APIGatewayProxyEventV2 | WarmerEvent;
 
 type AwsLambdaReturn = void;
+
+function formatWarmerResponse(event: WarmerEvent) {
+  const result = new Promise<WarmerResponse>((resolve) => {
+    setTimeout(() => {
+      resolve({ serverId, type: "warmer" } satisfies WarmerResponse);
+    }, event.delay);
+  });
+  return result;
+}
+
 const handler: WrapperHandler = async (handler, converter) =>
   awslambda.streamifyResponse(
     async (event: AwsLambdaEvent, responseStream): Promise<AwsLambdaReturn> => {
+      if ("type" in event) {
```

Comment:
Just a thought: This currently works, but it should have been designed as `event.type === 'warmer'` to be explicit.

Reply:
I think it should be `"type" in event && event.type === "warmer"`, otherwise typescript will complain